docs/source/management/role.rst

```diff
@@ -23,12 +23,16 @@ Operation defines the application logic action, for example::
 - write
 - order
 
+Note that in any of the above stanzas, a '*' is taked to mean 'all'.
+
 
 Resource Definitions
 ********************
-If an empty array is supplied for resource definitions then this is implied to mean all access. However, resource access can be limited by specifing an attribute filter.
+Resource Definitions are a somewhat trickier aspect of our implementation of RBAC. In general, ALL roles should be created with a resourceDefinitions stanza of '[]'. This is taken to mean "no additional filtering" and will generally result in the expected behavior. 
```

Comment:
Nice! Overall LGTM! But seems the description that resource definitions only apply to cost-management is missing here

Reply:
Ah, I wasn't sure whether to put that in specifically, I'll get it added!